src/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.spec.ts

```diff
@@ -1593,7 +1600,16 @@ describe('CaseFullAccessViewComponent - ends with caseID', () => {
           {provide: MatDialog, useValue: dialog},
           {provide: MatDialogRef, useValue: matDialogRef},
           {provide: MatDialogConfig, useValue: DIALOG_CONFIG},
-          DeleteOrCancelDialogComponent
+          DeleteOrCancelDialogComponent,
```

Comment:
@johnbenjamin-hmcts - Lines 1603 to 1612 look like new dependencies... did these come in as part of a merge or something?

Reply:
@Daniel-Lam - The test `describe('CaseFullAccessViewComponent - ends with caseID'...` was repeatedly failing due to these dependencies. This is because of referencing CaseEditComponent in FieldReadComponent.